.happo.js

```diff
@@ -0,0 +1,23 @@
+const { RemoteBrowserTarget } = require('happo.io')
+const happoPluginStorybook = require('happo-plugin-storybook')
+
+module.exports = {
+  apiKey: process.env.HAPPO_API_KEY,
+  apiSecret: process.env.HAPPO_API_SECRET,
+  targets: {
+    chrome: new RemoteBrowserTarget('chrome', {
+      viewport: '1024x768',
+    }),
+    edge: new RemoteBrowserTarget('edge', {
+      viewport: '1024x768',
+    }),
+    firefox: new RemoteBrowserTarget('firefox', {
+      viewport: '1024x768',
+    }),
```

Comment:
Happo supports a variety of browsers https://docs.happo.io/docs/browsers and we can also specify different view port sizes. I went with chrome, edge, and firefox at a reasonable desktop size for starting. Once we want mobile testing we can add some mobile configurations.